ewlegal-opinion.sty

```diff
@@ -1,23 +1,43 @@
-\ProvidesPackage{engwales-opinion}[2021-02-13 Package for pleading creation in England/Wales styles]
+%% TODO Stock beginning text, copyrights and project info and all that
 
-\RequirePackage[explicit]{titlesec}
-\RequirePackage{enumitem}
-\RequirePackage{fancyhdr}
+\ProvidesPackage{ewlegal-opinion}[2021-02-13 England and Wales Opinion Legal Template]
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typeset Elements
+%%%%%
 
 %% Formatting the main title
 \renewcommand\maketitle{%
   \begin{center}
     \bfseries
-    \renewcommand\arrayrulewidth{1.1pt}
-    \begin{tabular}{@{\hspace{12pt}}c@{\hspace{12pt}}}
-      \hline\\[-12pt]
-      \MakeUppercase{\@title}\\\ \\[-12pt]
-      \hline
-    \end{tabular}
+    \tramlined{\@title}
   \end{center}
   \medskip
 }
 
+%% Determine the spacing of the numbers in the point env
+% \renewcommand{\pointspacing}{leftmargin=0,labelsep=1cm}
+% TODO This work around to reusing the point env does not work. FIX!
+
+%% Create intro and facts commands for supposed convience
+\newcommand{\intro}{\section{INTRODUCTION}}
+\newcommand{\facts}{\section{FACTS}}
```

Comment:
do we really want to provide shortcuts like these? feels unnecessary. and it's a learning overhead to the user if they want to use the cls as intended. if we do this, we should make sure to provide almost all possible sections that can be in an opinion.

Reply:
I added them originally as more a demonstration of potential for @passamo9. The fact you can define and call arbitrary amounts of code like this.

Reply:
Removed.